src/Block/AuditBlockService.php

```diff
@@ -44,7 +44,7 @@ public function __construct($templatingOrDeprecatedName, object $templatingOrAud
         if ($templatingOrAuditReader instanceof EngineInterface) {
             @trigger_error(sprintf(
                 'Passing %s as argument 2 to %s() is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x'
-                .' and will throw a \TypeError in version 4.0. You must pass an instance of %s instead.',
+                .' and will throw a \TypeError exception in version 4.0. You must pass an instance of %s instead.',
```

Comment:
A type error is an error, not an exception: https://www.php.net/manual/en/class.typeerror.php

Reply:
I know `\Error` is sibling of `\Exception`, since both of them implements `\Throwable`, but since both can be thrown and catched, I tend to refer both as exceptions (regardless its class names).
To resume, maybe me reasoning can be expressed like this:

|Verb  |Noun       |
|---      |--             |
|trigger|error       |
|throw |exception|